### PR TITLE
Find openresty installed with homebrew on Mac

### PR DESCRIPTION
**Pull request by sharpobject**

### CODE REVIEW

**Comment by leafo** [comment]
Hmm, why aren't the specs running on this pull request. We recently switched over to github actions

**Comment by leafo** [comment]
Do you mind rebasing this PR, I think I fixed the spec issue

**Comment by sharpobject** [comment]
Alright.

File: lapis/cmd/nginx.moon [hunk 1]
```diff
@@ -94,11 +97,12 @@ class NginxRunner
         return @_nginx_path
 
     for prefix in *@nginx_search_paths
```
Comment (leafo) [inline lapis/cmd/nginx.moon:99]: can we swap how these loops are nested so that the existing search order is preserved with this change

Comment (sharpobject) [reply lapis/cmd/nginx.moon]: Sure yes, all the `nginx` then all the `openresty`

**Pull request merged**